docs/governance/templates/release_issue.md

```diff
@@ -44,37 +44,35 @@ and copy it into a release issue. Fill in relevant values, found inside {}
-    - [ ] If full release, copy `dev_supported_k8s` to `supported_k8s`.
-    - [ ] If full release, copy `dev_aks_minor_supported_k8s` to `aks_minor_supported_k8s`.
-    - [ ] If full release, copy `dev_minikube_minor_supported_k8s` to `minikube_minor_supported_k8s`.
-    - [ ] If full release, update documentation with updated example images tags.
+    - [ ] Copy the draft release content into a new `/site/content/en/blog/releases` content (this will be what you send via email).
+    - [ ] Review all `link_test` and `data-proofer-ignore` attributes and remove for link testing
+    - [ ] If full release, review and remove all instances of the `feature` shortcode
+    - [ ] If full release, add a link to previous version's documentation to nav dropdown.
+    - [ ] config.toml updates:
+        - [ ] If full release, update `release_branch` to the new release branch for {version}.
+        - [ ] If full release, update `release-version` with the new release version {version}.
+        - [ ] If full release, copy `dev_supported_k8s` to `supported_k8s`.
+        - [ ] If full release, copy `dev_aks_minor_supported_k8s` to `aks_minor_supported_k8s`.
+        - [ ] If full release, copy `dev_minikube_minor_supported_k8s` to `minikube_minor_supported_k8s`.
+        - [ ] If full release, update documentation with updated example images tags.
 - [ ] Create PR with these changes, and merge them with an approval.
-- [ ] Run `git remote update && git checkout main && git reset --hard upstream/main` to ensure your code is in line 
-   with upstream  (unless this is a hotfix, then do the same, but for the release branch)
+- [ ] Run `git remote update && git checkout main && git reset --hard upstream/main` to ensure your code is in line
+  with upstream  (unless this is a hotfix, then do the same, but for the release branch)
 - [ ] Publish SDK packages
-   - [ ] Run `make sdk-shell-node` to get interactive shell to publish node package. Requires Google internal process
-     to publish.
-   - [ ] Run `make sdk-publish-csharp` to deploy to NuGet. Requires login credentials. (if release candidate: 
-   `make sdk-publish-csharp RELEASE_VERSION={version}-rc`).
-   Will need [NuGet API Key](https://www.nuget.org/account/apikeys) from Agones account.
+    - [ ] Run `make sdk-shell-node` to get interactive shell to publish node package. Requires Google internal process
+      to publish.
+    - [ ] Run `make sdk-publish-csharp` to deploy to NuGet. Requires login credentials. (if release candidate:
+      `make sdk-publish-csharp RELEASE_VERSION={version}-rc`).
+      Will need [NuGet API Key](https://www.nuget.org/account/apikeys) from Agones account.
 - [ ] Run `make do-release`. (if release candidate: `make do-release RELEASE_VERSION={version}-rc`) to create and push the docker images and helm chart.
-- [ ] Run `make shell` and run `gcloud config configurations activate <your development project>` to switch Agones
```

Comment:
This line was removed - was that on purpose?

Reply:
Everything else looks good to me outside of this, so I think we can sneak this into the release? 😄

Reply:
It's removed accidentally. Added it back.